src/services/blocks/BlocksService.ts

```diff
@@ -340,8 +340,10 @@ export class BlocksService extends AbstractService {
 		block: Block
 	) {
 		const perByte = api.consts.transactionPayment?.transactionByteFee;
-		const extrinsicBaseWeight = api.consts.system
-			?.extrinsicBaseWeight as AbstractInt;
+		const extrinsicBaseWeight = api.consts.system.extrinsicBaseWeight
+			? (api.consts.system.extrinsicBaseWeight as AbstractInt)
+			: // We assume all dispatch classes have the same base_extrinsic value
+			  api.consts.system.blockWeights.perClass.normal.baseExtrinsic;
```

Comment:
so the js api has the value `api.consts.system.extrinsicBaseWeight` set before the weight refactor and for a chain that has it, the latter one?